backend/lib/calculate/adequacy.py

```diff
@@ -11,8 +12,7 @@
 
 from sqlalchemy.orm import sessionmaker
 
-# TODO - Use config or environment variable.
-MEASURER = distance.get_measure('haversine')
+MEASURER = distance.get_measure(config.get('measurer'))
 EXIT_DISTANCE = 10.0    # Measured in miles.
```

Comment:
Use clearer names to make the difference between EXIT_DISTANCE and RELEVANCY_RADIUS more apparent.

Reply:
This is not part of this PR.